src/neo-vm/StackItem.cs

```diff
@@ -10,7 +10,9 @@ namespace Neo.VM
 {
     public abstract class StackItem : IEquatable<StackItem>
     {
-        public static StackItem Null { get; } = new byte[0];
+        public bool IsNull => this is Null;
```

Comment:
This property is used only once, maybe we can remove it

Reply:
It is `public`. We can use it in neo.dll.